tomviz/OperatorPython.cxx

```diff
@@ -399,13 +399,21 @@ Operator* OperatorPython::clone() const
 
 bool OperatorPython::serialize(pugi::xml_node& ns) const
 {
+  ns.append_attribute("json_description")
+    .set_value(this->JSONDescription().toLatin1().data());
   ns.append_attribute("label").set_value(this->label().toLatin1().data());
   ns.append_attribute("script").set_value(this->script().toLatin1().data());
+  if (this->hasChildDataSource() && this->childDataSource()) {
+    pugi::xml_node childNode = ns.append_child("DataSource");
+    this->childDataSource()->serialize(childNode);
```

Comment:
Are these added to the ModuleManager's list of DataSources?  If so, they will be serialized twice.  Does that cause problems?  Will the serialized modules be deserialized in the right place?